new/vcloud-manager-blueprint.yaml

```diff
@@ -0,0 +1,876 @@
+        - type: DNAT
+          protocol: tcp
+          original_port:  { get_input: ssh_port }
+          translated_port: 22
+        - type: DNAT
+          protocol: tcp
+          original_port: 80
+          translated_port: 80
+        - type: DNAT
+          protocol: tcp
+          original_port: 443
+          translated_port: 443
+      vcloud_config: { get_property: [vcloud_configuration, vcloud_config] }
+
+  management_port:
+    type: cloudify.vcloud.nodes.Port
+    properties:
+      port:
+        network: { get_input: management_network_name }
+        ip_allocation_mode: { get_input: management_port_ip_allocation_mode }
+        ip_address: { get_input: management_port_ip_address }
+        primary_interface: true
+      vcloud_config: { get_property: [vcloud_configuration, vcloud_config] }
+    relationships:
+      - target: management_network
+        type: cloudify.vcloud.port_connected_to_network
+
+  manager_host_server:
+    type: cloudify.vcloud.nodes.Server
+    properties:
+      server:
+        name: { get_input: server_name }
+        catalog: { get_input: catalog }
+        template: { get_input: template }
+        hardware:
+          cpu: { get_input: manager_server_cpus }
+          memory: { get_input: manager_server_memory }
+        guest_customization:
+          computer_name: { get_input: server_name }
+      install_agent: false
+      management_network: { get_input: management_network_name }
+      vcloud_config: { get_property: [vcloud_configuration, vcloud_config] }
+    relationships:
+      - target: manager_public_ip
+        type: cloudify.vcloud.server_connected_to_public_nat
+      - target: ssh_keypair
+        type: cloudify.vcloud.server_connected_to_keypair
+      - target: management_port
+        type: cloudify.vcloud.server_connected_to_port
+      - target: manager_security_group
+        type: cloudify.vcloud.server_connected_to_security_group
+
+  manager_host:
+    type: cloudify.nodes.ManagerHost
+    relationships:
+      - type: cloudify.relationships.contained_in
+        target: manager_host_server
+    properties:
+      install_agent: false
+      ip: { get_attribute: [manager_host_server, ip] }
+      public_ip: { get_attribute: [manager_public_ip, public_ip] }
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        create:
+          implementation: fabric.fabric_plugin.tasks.run_script
+          inputs:
+            script_path: components/manager/scripts/vcloud/update-vcloud-vm.sh
+            fabric_env: &simple_fabric_env
+              user: { get_input: ssh_user }
+              key_filename: { get_input: ssh_key_filename }
+              host_string: { get_attribute: [manager_public_ip, public_ip] }
+              port: { get_input: ssh_port }
+              disable_known_hosts: True
+        configure:
+          implementation: fabric.fabric_plugin.tasks.run_task
+          inputs:
+            tasks_file: components/manager/scripts/vcloud/configure.py
```

Comment:
um, why is configure being run here instead of as part of the manager_configuration node? Again, you can take an example from the openstack manager blueprint. The configure script sets the provider context for the manager which is unrelated to its physical host but rather to its config.